hw1/02-palindrome.js

```diff
@@ -1,3 +1,30 @@
 const elem = document.querySelector('input');
+const response = document.getElementById('response');
 
-elem.addEventListener('input', handleInput);
+//function that checks if the number entered is a palindrome
+const handleInput = function handleInput() {
+
+    //quick way to not allow for negative integers or decimal values
+    if(elem.value)
+        elem.value = Math.floor(Math.abs(elem.value));
```

Comment:
This is a cool way to limit the number input! 

However, just from reading the code and comment, it was not immediately apparent to me that it is overriding the text inside the html input element on the actual webpage. I suggest tweaking the comment to mention that this modifies the value displayed to the user on the page as well.

Reply:
Fair point.